tests/versioned/common/pooling.js

```diff
@@ -11,18 +11,26 @@ const utils = require('@newrelic/test-utilities')
 
 module.exports = (t, requireMySQL) => {
   t.test('MySQL instrumentation with a connection pool', {timeout: 30000}, (t) => {
```

Comment:
refactored this to make it easier to await setup and not deal with wrapping the rest in an async function to properly handling the nested suites